core/src/oidc_auth.rs

```diff
@@ -40,9 +40,37 @@ pub enum Error {
     IssuerUriMustBeHttps,
     NoIdpServerInfo,
     CsrfMismatch,
+    MissingRefreshToken,
     Other(String),
 }
 
```

Comment:
I couldn't figure out the necessary types to make this a function, so much easier to have a macro that expands to a closure ;)